namedotcom/resource_namedotcom_record.go

```diff
@@ -3,6 +3,7 @@ package namedotcom
 import (
 	"strconv"
 	"strings"
+	"time"
```

Comment:
Use "golang.org/x/time/rate" here.
Also, we need to add limiter to all requests.